### DISCUSSION TITLE
Does Prometheus Operator and Alertmanager supports toleration ?

### DISCUSSION THREAD

**Discussion by munipravy** [prometheus-operator/prometheus-operator#3861] [Q&A]
**What did you do?**

**Did you expect to see some different?**

**Environment**

* Prometheus Operator version:

    `v41.0`
    

* Kubernetes version information:

    `1.17`
    

* Kubernetes cluster kind:

    insert how you created your cluster: kops, bootkube, etc.

* Manifests:

```
insert manifests relevant to the issue
```

* Prometheus Operator Logs:

```
 "error: error validating "/tmp/operator/alertmanager.yaml": error validating data: ValidationError(Alertmanager.spec.affinity): unknown field "tolerations" in com.coreos.monitoring.v1.Alertmanager.spec.affinity; if you choose to ignore these errors, turn validation off with --validate=false
"
```

**Does Prometheus Operator support pod affinity and tolerations?**:

**Comment by brancz** [comment]
Yes, see the API docs: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#alertmanagerspec
